File: src/main/java/io/github/dqualizer/dqlang/types_new/adapter/constants/LoadTestConstants.java
package io.github.dqualizer.dqlang.types_new.adapter.constants;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.github.dqualizer.dqlang.types_new.adapter.accuracy.Accuracy;
import io.github.dqualizer.dqlang.types_new.adapter.constants.loadprofile.LoadProfile;
import lombok.Getter;
import lombok.ToString;

/**
 * Java class for load test constants definitions
 */
@Getter
@ToString
public class LoadTestConstants {

    private Accuracy accuracy;
    @JsonProperty("load_profile")
    private LoadProfile loadProfile;
    @JsonProperty("response_time")
    private ResponseTime responseTime;
}

File: src/main/kotlin/dqualizer/dqlang/archive/k6adapter/dqlang/constants/LoadTestConstants.kt
package dqualizer.dqlang.archive.k6adapter.dqlang.constants

/**
 * Java class for load test constants definitions
 */
import com.fasterxml.jackson.annotation.JsonProperty
import dqualizer.dqlang.archive.k6adapter.dqlang.constants.accuracy.Accuracy
import dqualizer.dqlang.archive.k6adapter.dqlang.constants.loadprofile.LoadProfile
import lombok.Getter
import lombok.ToString

@Getter
@ToString
class LoadTestConstants {
    private val accuracy: Accuracy? = null

    @JsonProperty("load_profile")
    private val loadProfile: LoadProfile? = null

    @JsonProperty("response_time")
    private val responseTime: ResponseTime? = null
}
